plugins/jfrog-artifactory/src/api/artifactoryApi.ts

```diff
@@ -53,28 +52,25 @@ export class JfrogArtifactoryApiClient implements JfrogArtifactoryApiV1 {
     return await response.json();
   }
 
-  async getTags(repo: string) {
+  async getArtifact(repo: string) {
     const proxyUrl = await this.getBaseUrl();
-    const tagQuery = {
+    const packageQuery = {
       query:
-        'query ($filter: VersionFilter!, $first: Int, $orderBy: VersionOrder) { versions (filter: $filter, first: $first, orderBy: $orderBy) { edges { node { name, created, modified, package { id }, repos { name, type, leadFilePath }, licenses { name, source }, size, stats { downloadCount }, vulnerabilities { critical, high, medium, low, info, unknown, skipped }, files { name, lead, size, md5, sha1, sha256, mimeType } } } } }',
+        'query ($filter: PackageFilter!, $orderBy: PackageOrder) { packages (filter: $filter, orderBy: $orderBy) { edges { node { name, description, created, versions { name, size, repos { name }, package { packageType }, vulnerabilities { high, medium, low, info, unknown, skipped }, stats { downloadCount } } } } } }',
```

Comment:
```suggestion
        'query ($filter: PackageFilter!, $orderBy: PackageOrder) { packages (filter: $filter, orderBy: $orderBy) { edges { node { name, description, created, versions { name, size, repos { name }, package { packageType }, vulnerabilities { critical, high, medium, low, info, unknown, skipped }, stats { downloadCount } } } } } }',
```
`critical` is [undocumented](https://jfrog.com/help/r/jfrog-rest-apis/graphql), but present. Must also be applied in other parts of this patch.

Reply:
Ah you are right sorry 😄 will update this.